modules/convert/git_commit.go

```diff
@@ -73,7 +73,7 @@ func ToPayloadCommit(repo *repo_model.Repository, c *git.Commit) *api.PayloadCom
 }
 
 // ToCommit convert a git.Commit to api.Commit
-func ToCommit(repo *repo_model.Repository, gitRepo *git.Repository, commit *git.Commit, userCache map[string]*user_model.User) (*api.Commit, error) {
+func ToCommit(repo *repo_model.Repository, gitRepo *git.Repository, commit *git.Commit, userCache map[string]*user_model.User, stat bool) (*api.Commit, error) {
```

Comment:
Could you please document the new parameter?
Otherwise, it will confuse people looking over the code.

Reply:
Hmm, missed the comment and merged (I saw the comment in code "Retrieve files affected by the commit") 😂

Maybe next time this function could have more comments